src/styles/project.st.css

```diff
@@ -0,0 +1,135 @@
+/**
+ * Shelley Project - project.st.css
```

Comment:
I just copied the Shelley project and adapted that, you can call this what you want or create a shared one that you could use between projects.